tasks/parse-pdr/schemas/config.json

```diff
@@ -8,13 +8,26 @@
       "type": "string"
     },
     "stack": {
-      "description": "The name of the Task's CloudFormation Task, useful as a prefix",
+      "description": "The cumulus stack prefix, pulled from $.meta.stack",
       "type": "string"
     },
     "granuleIdFilter": {
       "description": "A regular expression that is applied against the filename of the first file of each granule contained in the PDR.  If the regular expression matches, then the granule is included in the output.  Defaults to '.', which will match all granules in the PDR.",
       "type": "string"
     },
+    "uniquifyGranuleId": {
```

Comment:
The boolean config parameter in other tasks don't have to support both string and bean types. Why do we need to support here.

Reply:
Largely because it's more user friendly, and users have frequently run into this issue - is there a good reason not to?